checkov/terraform/graph_builder/variable_rendering/renderer.py

```diff
@@ -80,7 +80,10 @@ def attributes_no_eval(self, attribute: str, vertex_index: int) -> bool:
     def evaluate_vertex_attribute_from_edge(self, edge_list: List[Edge]) -> None:
         multiple_edges = len(edge_list) > 1
         edge = edge_list[0]
-        origin_vertex_attributes = self.local_graph.vertices[edge.origin].attributes
+        try:
+            origin_vertex_attributes = self.local_graph.vertices[edge.origin].attributes
+        except AttributeError:
```

Comment:
why would it raise an attribute error here? (asking more to understand than adding a review comment)

Reply:
In the subgraph, we are adding a non/{} vertices.